### PR TITLE
update spec to match oog call circuit

### PR DESCRIPTION
**Pull request by DreamWuGit**
-  update to read tx account access list instead of write , circuit PR https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/827

### CODE REVIEW

File: tests/evm/test_oog_call.py [hunk 1]
```diff
@@ -228,9 +223,7 @@ def test_oog_call_not_root(caller_ctx: CallerContext, callee: Account):
             .stack_read(2, 1022, RLC(stack.rd_offset, randomness))
             .stack_read(2, 1023, RLC(stack.rd_length, randomness))
             .stack_write(2, 1023, RLC(False, randomness))
-            .tx_access_list_account_write(1, callee.address, True, is_warm_access,
-                                          rw_counter_of_reversion=caller_rw_counter_end_of_reversion - caller_ctx.reversible_write_counter)
-            .account_read(callee.address, AccountFieldTag.Balance, callee_balance)
+            .tx_access_list_account_read(1, callee.address, is_warm_access)            .account_read(callee.address, AccountFieldTag.Balance, callee_balance)
```
Comment (han0110) [inline tests/evm/test_oog_call.py:226]:
```suggestion
            .tx_access_list_account_read(1, callee.address, is_warm_access)
            .account_read(callee.address, AccountFieldTag.Balance, callee_balance)
```

**Review by han0110 — APPROVED** [review]
LGTM!

**Comment by ed255** [comment]
@icemelon could you find a second review for this PR from scroll?  Thanks!

**Review by silathdiir — APPROVED** [review]
LGTM

**Pull request merged**